### PR TITLE
Multiple addresses proposal

### PR DESCRIPTION
**Pull request by jenssegers**

### CODE REVIEW

**Comment by stivni** [comment]
still a bit doubtful about this one... as we don't have all fields for all kinds of addresses today...

**Comment by WouterSioen** [comment]
@stivni @jenssegers @jdrieghe @Brammm Will this be merged? We're working on the companies.info endpoint, so it could be useful to implement it this way if we want this to be merged. /cc @vanbrabantf

File: apiary.apib [hunk 1]
```diff
@@ -1703,18 +1685,18 @@ Register a webhook.
...
-+ country: `US` (string)
-
-## Addressee (object)
++ type: `business` (enum[string], required)
+    + Members
+        + business
+        + invoice
+        + delivery
+        + visiting
+        + other
```
Comment (mrnnn) [inline apiary.apib]: Do we need "default"? Or is that "business". We don't have any option today to handle "other", so leave that out.

File: apiary.apib [hunk 2]
```diff
@@ -1703,18 +1685,18 @@ Register a webhook.
...
 + addressee: `Pied Piper Finance Dep.` (string, optional)
-+ street: `Newell Road` (string)
-+ number: `5230` (string)
-+ zipcode: `94020` (string)
++ line1: `5230 Newell Road` (string)
```
Comment (mrnnn) [inline apiary.apib]: We don't have addressee for the default address. That could be a bit of a deal breaker here.

File: apiary.apib [hunk 3]
```diff
@@ -1703,18 +1685,18 @@ Register a webhook.
...
++ line2: `2nd floor` (string, optional)
 + city: `Palo Alto` (string)
++ postal_code: `94020` (string)
```
Comment (mrnnn) [inline apiary.apib]: I like

**Comment by jenssegers** [comment]
Let's split up this PR into smaller PR's. Part #1 at https://github.com/teamleadercrm/api/pull/182

**Comment by stivni** [comment]
closing because of smaller PRs

**Pull request closed without merging**